edenia/antelope-tools
# Feat/improve UI faucet

### Improve UI faucet of Libre Testnet

### What does this PR do?

- Resolve #1258 

### Steps to test

1. Run the project with Libre testnet configuration
2. Go to faucet page
3. Check the forms and the texts

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/39aa9c68-1a12-4fdd-86d6-b3433b596b63)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/0618f99f-410e-48be-9cb0-f24f767680ca)
